lib/gui/app/modules/analytics.ts

```diff
@@ -119,4 +219,11 @@ export function logEvent(message: string, data: _.Dictionary<any> = {}) {
  * @description
  * This function logs an exception to error reporting services.
  */
-export const logException = resinCorvus.logException;
+export function logException(error: any) {
+	const shouldReportErrors = settings.getSync('errorReporting');
+	if (shouldReportErrors) {
+		initAnalytics();
+		console.error(error);
+		SentryRenderer.captureException(error);
+	}
```

Comment:
...and same point here about errors being visible to the user regardless of if they're being sent to Balena or not.

Reply:
good point; same answer

Reply:
made it part of this PR : https://github.com/balena-io/etcher/pull/3978/commits